core/bind/core_bind.cpp

```diff
@@ -1142,11 +1142,11 @@ void _OS::_bind_methods() {
 	BIND_ENUM_CONSTANT(SYSTEM_DIR_PICTURES);
 	BIND_ENUM_CONSTANT(SYSTEM_DIR_RINGTONES);
 
-	BIND_ENUM_CONSTANT(POWERSTATE_UNKNOWN);
-	BIND_ENUM_CONSTANT(POWERSTATE_ON_BATTERY);
-	BIND_ENUM_CONSTANT(POWERSTATE_NO_BATTERY);
-	BIND_ENUM_CONSTANT(POWERSTATE_CHARGING);
-	BIND_ENUM_CONSTANT(POWERSTATE_CHARGED);
+	BIND_ENUM_CONSTANT(OS::POWERSTATE_UNKNOWN);
+	BIND_ENUM_CONSTANT(OS::POWERSTATE_ON_BATTERY);
+	BIND_ENUM_CONSTANT(OS::POWERSTATE_NO_BATTERY);
+	BIND_ENUM_CONSTANT(OS::POWERSTATE_CHARGING);
+	BIND_ENUM_CONSTANT(OS::POWERSTATE_CHARGED);
```

Comment:
Sorry, didn't notice until now. This is wrong, the argument you pass to `BIND_CONSTANT` and `BIND_ENUM_CONSTANT` is stringified, therefore you cannot include `OS::` in it.

Reply:
Oh, should I revert or can you properly fix it?

Reply:
I am having a look at it right now.

Reply:
Looks like the way other classes do this is simply by declaring the same enum in `_OS` too. You can have a look at `SaverFlags` in `ResourceSaver` and `_ResourceSaver` as an example (BTW, the latter is missing the `FLAG_REPLACE_SUBRESOURCE_PATHS` constant).

cc @maxim-sheronov

Reply:
Here it is #11316